### ISSUE TITLE
Does PGGB miss odgi sort -O?

### ISSUE THREAD

**Issue by subwaystation** [pangenome/pggb#111]
From @brettChapman https://github.com/pangenome/odgi/issues/282

> Hi @subwaystation and @ekg 
Something I thought worth pointing out, while running odgi I came across errors about my graphs being unsorted, so I've added 
odgi sort with the -O flag prior to odgi viz to my script. I notice odgi sort is left out of the PGGB script, which may cause issues with some graphs, like I've found with mine after removal of the consensus paths. It might be worth adding that into the PGGB script to avoid possible issues with graphs.

**Comment by subwaystation** [comment]
@brettChapman 
Where there errors while you ran PGGB with a certain combination of parameters? Or was this independent of a PGGB run?
Could you please write the commands you used? Or give an example with a minimal data set?
Thanks!

**Comment by brettChapman** [comment]
Hi @subwaystation 

These were errors I received after I had completed the PGGB run. It happened with some graphs but not all.

After I generated the smoothed GFA, they contained consensus paths (an earlier build of PGGB before consensus paths were dropped by default). I proceeded to remove the consensus paths and was left with VG graphs without the consensus. I then converted these to GFA so I could then build OG graphs with ODGI and visualise the 2D graphs without the consensus paths. Some ODGI runs complained that the graph was unsorted and suggested I run ```odgi sort``` with -O. See an error below for my 3H graph.

```
odgi viz -i barley_pangenome_graph_3H.gfa.og -o barley_pangenome_graph_3H.gfa.og.viz_mqc.png -x 1500 -y 500 -P 10 -b
[odgi::viz] error: the graph is not optimized. Please run 'odgi sort' using -O, --optimize
```

**Comment by subwaystation** [comment]
How did you remove the consensus graphs? 
Note that `odgi viz` has the `-I, --ignore-prefix` option, which would allow you to ignore paths with a certain prefix.

**Comment by brettChapman** [comment]
Hi @subwaystation

I needed to remove the consensus paths because they were getting in the way when using sequenceTubeMap. The plan was to visualise interactively in sequenceTubeMap for our end users (barley breeders and geneticists). Lots of consensus paths would just confuse the visualisation. I ended up running vg chunk on the entire graph to get gene chunks into sequenceTubeMap because it currently can not visualise the whole pangenome graph at once.

I ran this code to remove the consensus paths:
```
vg convert -g test_graph.gfa -p > test_graph.pg
vg index test_graph.pg -x test_graph.xg
vg paths -x test_graph.xg -L | grep Consensus > consensus_paths.txt
vg paths -v test_graph.pg -p consensus_paths.txt -d > test_graph.noConsensus.vg
vg index test_graph.noConsensus.vg -x test_graph.noConsensus.xg
```
